modules/web3/constants/chainID.ts

```diff
@@ -9,8 +9,8 @@ SPDX-License-Identifier: AGPL-3.0-or-later
 export enum SupportedChainId {
   MAINNET = 1,
   GOERLI = 5,
-  GOERLIFORK = 31337,
   ARBITRUM = 42161,
   ARBITRUMTESTNET = 421613,
-  ARBITRUMTESTNETFORK = 521613
+  ARBITRUMTESTNETFORK = 521613,
+  TENDERLY = 314310
```

Comment:
Wondering if the Tenderly and Arbitrum testnet fork IDs should be changed to `31400001` and `31442161` since those are the IDs that were used to create the Tenderly forks (and I think those are the ones that are used by the Spock instances)

![image](https://github.com/makerdao/governance-portal-v2/assets/22449631/69f41ae8-4e22-435d-8f6d-911bb5d1d20c)
![image](https://github.com/makerdao/governance-portal-v2/assets/22449631/1127f5d7-57fc-42db-b68b-639465c79021)